src/yargs-commands/eval.mjs

```diff
@@ -183,7 +183,7 @@ async function doEval(argv) {
       version: argv.version,
       argv
     })).client
-    : await (container.resolve("getSimpleClient")(argv))
+    : container.resolve("getSimpleClient")(argv)
```

Comment:
awaiting the container resolution isn't necessary - i cleaned it up where i found it